### PR TITLE
Installation instructions

### PR DESCRIPTION
**Pull request by tfiers**
See here for what it looks like: https://github.com/tfiers/cosyne-tutorial-2022#readme

I also added an intro paragraph to the readme

### CODE REVIEW

**Comment by tfiers** [comment]
@rdgao, I added some stuff for MacOS (and Ubuntu), but I don't have a mac (or a working MacOS .iso image) to test it.
So if you could read through [the instructions](https://github.com/tfiers/cosyne-tutorial-2022#run-locally-), and if they're all right, then @thesamovar I think this is good to merge

**Comment by thesamovar** [comment]
I'll wait for @rdgao to comment and then merge.

**Comment by tfiers** [comment]
Confirmed that the installation instructions are succesful on a fresh Ubuntu (LTS, 20.04) and a fresh Windows 10 Home install.
Only tested _brian and the first two notebooks; not yet _torch.
I learned that `jupyter notebook` does not work in PowerShell; I'll update the readme to reflect that.

**Comment by rdgao** [comment]
made some edits, mostly minor additions to make things more explicit, but one important change was: 
`cd C:\Users\jane\Desktop\cosyne-tutorial-2022` instead of `cd cd C:\Users\jane\Desktop\cosyne-tutorial-2022-main`. I don't know if this is a specific thing for when you download via zip, but if you clone from command line, the former is the right directory.

otherwise, both environments install fine on MacOS 11.1.

**Comment by tfiers** [comment]
Excellent, thanks for the improvements.

If downloading a zip, it indeed appends the branch name (`-main`). I'll clarify in the readme.

File: README.md [hunk 1]
```diff
@@ -148,3 +151,9 @@ In more detail (click on a section to expand it):
 - When you are done with the notebooks, you can exit the notebook server application
   that is still running in your terminal with `Ctrl`-`C`
 </details>
+
+<details>
+<summary>6. <b>Repeat for Exercise Part 2</b></summary>
+
+- For Part 2 of the tutorial (using pytorch), repeat steps 4 & 5, but replace `cosyne22_brian` with `cosyne22_torch`.
```
Comment (tfiers) [inline README.md]: also the `env create -f` file; I'll update

**Comment by tfiers** [comment]
voila, ready to merge

**Comment by thesamovar** [comment]
Many thanks!

**Pull request merged**